### PR TITLE
Fixing Python tornadoes test for Google

### PR DESCRIPTION
**Pull request by pabloem**
The tornadoes test, when run from DirectRunner needs a GCS location. When running on Dataflow, it should just use the GCS-based temp location of the pipeline.

r: @markflyhigh

### CODE REVIEW

**Comment by pabloem** [comment]
Run Python PostCommit

File: sdks/python/apache_beam/examples/cookbook/bigquery_tornadoes.py [hunk 1]
```diff
@@ -87,14 +87,19 @@ def run(argv=None):
     rows = p | 'read' >> beam.io.Read(beam.io.BigQuerySource(known_args.input))
     counts = count_tornadoes(rows)
 
+    if p.options.get_all_options()['temp_location'].startswith('gs://'):
```
Comment (markflyhigh) [inline sdks/python/apache_beam/examples/cookbook/bigquery_tornadoes.py]: Seems temp_location has to be a GCS path (https://github.com/apache/beam/blob/master/sdks/python/apache_beam/options/pipeline_options.py#L426). May not need `gs://` check.

**Comment by pabloem** [comment]
Fixed that. Thanks Mark.

**Comment by pabloem** [comment]
Run Python PostCommit

**Comment by markflyhigh** [comment]
LGTM to me. Thanks

**Pull request merged**